frontend/src/components/Account/common/ProfileHistory/ProfileHistory.scss

```diff
@@ -1,7 +1,7 @@
 .profile-history-container {
     margin-bottom: 0px;
     .profile-history-text {
-        margin: 44px 0px 0px 0px;
+        margin: 44px 0px;
```

Comment:
Do we want `margin-bottom: 44px` here? I remember being a bug in the past to remove it.

Reply:
Yes, we do. At least, according to https://github.com/code4romania/stam-acasa/issues/393

Reply:
Here is the original modifications: https://github.com/code4romania/stam-acasa/pull/356/files made for this issue #304

Reply:
Hmm. ok. I can see that issue it's related to mobile spacing. Will check it out later tonight and make sure we only have the new changes applied for desktop.

Reply:
Checked it and looks ok on mobile. The previous styles were for `.profile-history-container`. IMO the spacing is fine here, but thanks for mentioning it 👌 
![Screenshot 2020-04-29 at 18 07 51](https://user-images.githubusercontent.com/1295800/80619016-67141500-8a44-11ea-84a9-6ea98bf223c0.png)